terraform/examples/multi-node-cluster/README.md

```diff
@@ -0,0 +1,7 @@
+# 2 Node Cluster Example
```

Comment:
Seems like this README may need updating?

Reply:
Doh! Good catch.